src/App.css

```diff
@@ -92,8 +177,152 @@ body {
   margin: 1rem auto;
 }
 
-.eachbook {
+.bookcard {
   display: flex;
   justify-content: space-between;
-  padding: 1rem;
+  border: 1px solid #e8e8e8;
+  margin: 6.25rem;
+  background-color: #fff;
+  padding: 2rem;
+  flex: 1;
+}
+
+.bookCategory {
+  font-size: 0.875rem;
+  color: #888;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.bookTitle {
+  font-family: 'Roboto Slab', serif;
+  font-weight: 900;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+}
+
+.bookAuthor {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  color: #4386bf;
+  font-weight: 300;
+}
+
+.bookActions {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  color: #4386bf;
+  font-weight: 300;
+  margin-right: 0.5rem;
+  cursor: pointer;
+}
+
+.delete {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  color: #4386bf;
+  font-weight: 300;
+  margin-right: 0.3rem;
+  margin-left: 0.2rem;
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.actions {
+  margin-top: 0.5rem;
+}
+
+.rightBorder {
+  border-right: 2px solid #e8e8e8;
+  padding: 2px 0 2px 0;
+}
+
+.divider {
+  border-right: 2px solid #e8e8e8;
+  padding: 2px 0 2px 0;
+}
+
+.edit {
+  margin-left: 0.5rem;
+}
+
+.progress {
+  display: flex;
+}
+
+.progReport {
+  padding-top: 1.5rem;
+  padding-left: 0.5rem;
+}
+
+.propPercent {
+  font-size: 2rem;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 300;
+}
+
+.rate {
+  font-size: 0.875rem;
+  color: #888;
+}
+
+.chapter h2 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: 300;
+  color: #888;
+}
+
+.chapter h3 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: 300;
+}
+
+.progButton {
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  margin-top: 1rem;
+  padding-left: 2rem;
+  padding-right: 2rem;
```

Comment:
- [ ] Instead of using padding-top, padding-right, padding-bottom, and padding-left separately you can simply use just one line: `padding: 0.5 2rem`;